### ISSUE TITLE
ACE CR - 24/12/2015

### ISSUE THREAD

**Issue by DHANDAPANISATTANATHAN** [SSOMENSREPO/ACE-AUTOLUTION#17]
@ ACC TEAM,

TARGET : 24/12/2015

POINT 4 - TRY TO CLOSE 26/12/2015

1. Change disc in % nw nt amount based.Change as per all  - SKL

2.Also in ADD JS, rename mobile  no to contact no. - SAFI

3.what they enter in ADD CUST contact no ,shd sync to contact no in ADD JS form - SAFI
CHASSIS no -> not mandatory
DELETE CAR NO plate field , remove in inv pdf also

4. Each individual inv - profit and loss, like buying price, - selling price = profit/loss = Report - SKL

5.font size reduce in pdf inv, pdf in one page min 8 products - AI

6.in CUSTOMER DT remove CUST ID HDR ,ADD REG NO HDR. - BC & SAFI
in ADD JS -> ADD CUST
replace CUST COMPANY REG NO by just REG NO which ll be equal to the REG NO from ADD JS form both the REG NO ll be equal

7. Reduce the ace logo (squeeze the top wise now its too high, it shd be lower )size in pdf - Urgent

**Comment by DHANDAPANISATTANATHAN** [comment]
8. DEF - Purchase price shld come in purchase now selling price came - SAFI

9. PRODUCT - SKU CUSTOMER ND SKU VENDOR remove from form. - KK

 but in DB auto generate number like SKUC-00001,... ND SKUV-00001,...

**Comment by DHANDAPANISATTANATHAN** [comment]
10. BALANCE SHEET - SP SIDE ISSUE WRONGLY SENDING  DEBIT AND CREDIT -SQL TEAM

**Comment by DHANDAPANISATTANATHAN** [comment]
# RE OPEN: BC

Why customer setting form.still Reg no and main office number label not changed.

Contact No - change like this 'Contact Number' in JS cust pop up.

**Comment by DHANDAPANISATTANATHAN** [comment]
2. Chasis no non mandatory. but in edit form it showing mandatory - SAFI

**Comment by DHANDAPANISATTANATHAN** [comment]
2. Less Payment to be renamed to Less Discount in Inv pdf - AI

**Comment by arthisena** [comment]
status*:done*



*Regards,*

*ARTHI.K**JUNIOR WEB DEVELOPER*
*​**SSOMENS SOFTWARE DEVELOPMENT*
*04132333229 /230/231*
www.ssomens.com
[image: C:\Users\SSOMENS-003_2\Desktop\business development
material\customLogo.gif.png]

**Comment by safiyullahmohideen** [comment]
Above issue in JS fixed and uploaded in ace.

Then corrected unit price validation in purchase form  and uploaded

Status:Fixed
